### DISCUSSION TITLE
Editieren von Polygonen

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#167] [Q&A]
Gibt es eine Obergrenze von Vertices, ab der Polygone nicht mehr im webGIS bearbeitet werden können?

Im webGIS werden größerer Gebiete (generalisierte Gemeinden) erfasst:
- Aus Objektgeometrie übernehmen
- Merge ![image](https://github.com/gis-eni/webgis-forum/assets/1842500/5c0aebf2-5adb-4b12-b49b-3063e280e21b)

Da diese zu erfassenden Gebiete nicht exakt den Gemeindegrenzen entsprechen, müssen die Gebietsgrenzen da und dort bearbeitet werden.
Bei ca. 2.000 Vertices beträgt die Ladezeit ca. 2 Minuten. Es ist aber nicht das Polygon mit den meisten Vertices. 
Gibt es eine Möglichkeit die Ladezeit zu verkürzen?

**Comment by gis-eni** [comment]
Dazu kann ich leider nix genaues sagen. Was bedeutet "Ladezeit"? Wo ist hier der Flaschenhals...
Ich ich das einmal testen?

**Reply by marko-jer** [reply]
Ladezeit: ich wähle ein Objekt zur Bearbeitung aus ... danach dauert es lange, bis im webGIS weitergearbeitet werden kann. Mit der Browsermeldung 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/43b78d9b-02e8-4644-ae0b-89f75572cde1)
Wo der Flaschenhals ist kann ich nicht beurteilen.
Der Testzugang ist per Email unterwegs!

**Comment by gis-eni** [comment]
Ich konnte das Problem nachvollziehen. Das Testpolygon, dass du mir geschickt hast ging zwar zum editieren. Allerdings dauert das hinzufügen eines Vertex ca 8 Sekunden (oder auch beim erstmaligen Anklicken):

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/3d3f3911-59ad-4d88-b60f-69ab57320f39)

Polygone mit mehreren Tausend Vertices (Gemeinde Graz, Katastergenau gingen gar nicht mehr. Punkt hinzufügen dauert > 30 Sekunden...


Ich habe versuche ein paar Optimierungen anzubringen bin mit der Response Zeit deutlich runter gekommen:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/f907bb35-4338-4da3-9e0b-555d08eb6c5e)

Unter einer halben Sekunde, finde ich zum Arbeiten noch ok (hängt natürlich immer von der Hardware ab. Das läuft alles im Browser)

Damit kann man theoretisch auch Graz katastergenau editieren. Da dauert halt das hinzüfgen eines Vertex bei mir ca 4 Sekunden.... ;) aber es geht.

Ich möchte da aber noch ein bisserl Testen, bevor das so freigeben.
